tests/integration/mock_auth.py

```diff
@@ -114,28 +124,117 @@ async def jwk_response(request: web.Request) -> web.Response:
 
 async def userinfo(request: web.Request) -> web.Response:
     """Mock an authentication to ELIXIR AAI for GA4GH claims."""
-    global nonce, user_eppn, user_family_name, user_given_name
+    global nonce, user_sub, user_family_name, user_given_name
     user_info = {
-        "sub": "smth",
         "eduPersonAffiliation": "member;staff",
-        "eppn": user_eppn,
+        "sub": user_sub,
         "displayName": f"{user_given_name} {user_family_name}",
         "schacHomeOrganizationType": "urn:schac:homeOrganizationType:test:other",
         "given_name": user_given_name,
-        "uid": user_eppn,
+        "uid": user_sub,
         "nsAccountLock": "false",
         "eduPersonScopedAffiliation": "staff@test.what;member@test.what",
         "name": f"{user_given_name} {user_family_name}",
         "schacHomeOrganization": "test.what",
         "family_name": user_family_name,
-        "email": user_eppn,
+        "email": user_sub,
     }
 
     logging.info(user_info)
 
     return web.json_response(user_info)
 
 
+async def oidc_config(request: web.Request) -> web.Response:
+    """Return standard OIDC configuration."""
+    oidc_config_json = {
+        "issuer": "http://mockauth:8000",
+        # must be localhost to be accessible outside of docker-network
+        "authorization_endpoint": "http://localhost:8000/authorize",
+        "token_endpoint": "http://mockauth:8000/token",
```

Comment:
`http://mockauth:8000` maybe it can be a parameter ? the same for `localhost:8000`

Reply:
parameterised URLs